.circleci/config.yml

```diff
@@ -56,7 +56,7 @@ commands:
           name: Reset package-lock
           command: 'git checkout -- package-lock.json'
       - save_cache:
-          key: v22-npm-{{ checksum "package.json" }}-{{ checksum "package-lock.json" }}
```

Comment:
💭 Just thinking about this, but maybe we should revisit when and why we bump this version. To my memory, I think as long as the `package-lock` is actually different due to a difference in updated packages, we shouldn't need to bump the cache key every round of updates. It should automatically use the new version since the checksums will be different. If we want to force it to use a new cache key despite the package-lock's checksum not changing then we have to bump it up.

Reply:
That's a good point, I was just blindly doing this because the docs said to; I'll test and update the docs accordingly

Reply:
Thank you!